pkg/counter/cache_test.go

```diff
@@ -160,7 +162,11 @@ func TestInitializeCounterFromToolchainClusterWithNegativeNumbersInCache(t *test
 	AssertThatGivenToolchainStatus(t, toolchainStatus).
 		HasMurCount(12).
 		HasUserAccountCount("member-1", 9).
-		HasUserAccountCount("member-2", 3)
+		HasUserAccountCount("member-2", 3).
+		HasUsersPerActivations(map[string]int{
+			"1": 0,
+			"2": 1,
+		})
 }
 
 func TestInitializeCounterByLoadingExistingResources(t *testing.T) {
```

Comment:
you didn't modify the unit test that verifies loading from existing resources - should you create some UserSignups as part of the test setup?

Reply:
done in 0489c18

Reply:
In the commit that you referenced, you didn't modify anything in the `TestInitializeCounterByLoadingExistingResources` test - you didn't add any UserSignups. Sorry, if I missed anything

Reply:
that's because "from existing resources", I thought you meant: initializing from an existing ToolchainStatus resource

Reply:
addressed in cd98632